exercises/practice/scrabble-score/.meta/config.json

```
@@ -1,6 +1,14 @@
 {
   "blurb": "Given a word, compute the Scrabble score for that word.",
-  "authors": [],
+  "authors": [
+    "jvarness"
+  ],
+  "contributors": [
+    "devkabiir",
+    "ErikSchierboom",
+    "jackhughesweb",
+    "Stargator"
+  ],
```

```suggestion
  "contributors": [
  ],
```

---

I suggest adding my name since I did revisions to the exercise that resulted in either adding/removing test cases or modifying them due to syncing the exercise to the spec.

```suggestion
  "contributors": [
    "Stargator"
  ],
```